auth/http.go

```diff
@@ -1405,6 +1430,57 @@ func httpHandlerGetGroups(central *ImqsCentral, w http.ResponseWriter, r *httpRe
 	}
 }
 
+func httpHandlerExportUserGroups(central *ImqsCentral, w http.ResponseWriter, r *httpRequest) {
+
+	users, err := central.Central.GetAuthenticatorIdentities(authaus.GetIdentitiesFlagNone)
+	if err != nil {
+		authaus.HttpSendTxt(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+
+	ident2perm, err := central.Central.GetPermits()
+	if err != nil {
+		authaus.HttpSendTxt(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+
+	groups, err := central.Central.GetRoleGroupDB().GetGroupsRaw()
+	if err != nil {
+		authaus.HttpSendTxt(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+
+	groupsResp, err := getPermitsJSON(central, users, ident2perm)
+	if err != nil {
+		authaus.HttpSendTxt(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+
+	var exportGroups []exportGroup
+	groupNametoID := make(map[string]int)
+	for _, group := range groups {
+		exportGroups = append(exportGroups, exportGroup{Id: int(group.ID), Name: group.Name, Permlist: group.PermList})
+		groupNametoID[group.Name] = int(group.ID)
+	}
+
+	var exportGroupUsers []exportGroupUser
+
+	for _, user := range groupsResp {
+		groups := make([]int, len(user.Groups))
+		for i, group := range user.Groups {
+			groups[i] = groupNametoID[group]
+		}
+		if user.Email != "" {
+			exportGroupUsers = append(exportGroupUsers, exportGroupUser{ID: "email: " + user.Email, Groups: groups})
+		} else if user.UserName != "" {
+			exportGroupUsers = append(exportGroupUsers, exportGroupUser{ID: "username: " + user.UserName, Groups: groups})
+		}
+	}
+
+	var userGroupsJson = &userGroups{Users: exportGroupUsers, Groups: exportGroups}
+	httpSendJson(w, userGroupsJson)
```

Comment:
No need to instantiate this variable, just pass the newly initialized object to be sent:
```suggestion
	httpSendJson(w, userGroups{Users: exportGroupUsers, Groups: exportGroups})
```